staging/src/k8s.io/apiserver/pkg/server/filters/reqmgmt.go

```diff
@@ -48,12 +47,14 @@ func WithRequestManagement(
 		ctx := r.Context()
 		requestInfo, ok := apirequest.RequestInfoFrom(ctx)
 		if !ok {
-			handleError(w, r, fmt.Errorf("no RequestInfo found in context"))
+			klog.Errorf("No RequestInfo found in context %#+v of request %#+v", ctx, r)
```

Comment:
https://github.com/kubernetes/kubernetes/blob/cd5bc3c383b0359029aa82b18828971622cca210/staging/src/k8s.io/apiserver/pkg/server/deprecated_insecure_serving.go#L88-L91

is the change valid? i see other filters does rejections on failing to find request-info/user from ctx..

Reply:
(A) If we think the apiserver will not produce this condition, but it does anyway, this is not an error in the request and so the request should not be rejected as an error.

(B) See https://github.com/kubernetes/kubernetes/pull/81224#issuecomment-521806266 for an example showing that there is indeed no userInfo for a request received through an unsecured port (which was the case in that instance).

Reply:
> (B) See #81224 (comment) for an example showing that there is indeed no userInfo for a request received through an unsecured port (which was the case in that instance).

is the system completely started when you use kubectl? i.e. is `/healthz` returning `ok`? i remember it happens sometimes when we requests an apiserver not completely launched..

Reply:
(A) If we think the apiserver will not produce this condition, but it does anyway, this is not an error in the request and so the request should not be rejected as an error.

(B) I will re-investigate for our information, but it does not matter here.  Our filter should not introduce request rejections in situations where requests are not currently rejected.

Reply:
I agree we should not introduce new request rejections cases, but should we still do rate limiting when request info or user info is missing? i think we should. WDYT?

Reply:
i think the error is suggesting sth is wrong w/ the insecure handler chaining, not the req-mgmt handler to blame.. i am slightly leaning on a fix from the master